packages/gatsby/src/query/error-parser.js

```diff
@@ -24,6 +24,22 @@ const errorParser = ({ message, filePath, location }) => {
         }
       },
     },
+    // Warn users about a possible Windows-specific error resulting from uppercase folders:
+    // "Error: RelayParser: Encountered duplicated definitions for one or more documents:
+    // each document must have a unique name. Duplicated documents:"
+    {
+      regex: /Error: RelayParser: Encountered duplicated.*/gm,
+      cb: match => {
+        return {
+          id: `85902`,
+          context: {
+            sourceMessage:
+              match[0] +
+              `\n\nWarning: Windows users may be able to fix this error by renaming all directories to lowercase\nin the file path and also by installing package node-win-env using npm install -g node-win-env`,
```

Comment:
I'm not sure what `node-win-env` would fix here? `node-win-env` fixes env-variables syntax. I can't find in their readme that it fixes this error.